modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/components/field-types/FieldTypeList.es.js

```diff
@@ -97,7 +96,16 @@ const FieldTypeList = ({
 		.sort(({displayOrder: a}, {displayOrder: b}) => a - b);
 
 	if (showEmptyState && !filteredFieldTypes.length) {
-		return <EmptyState emptyState={emptyState} keywords={keywords} small />;
+		return (
+			<ClayEmptyState
+				description={Liferay.Language.get(
+					'try-again-with-a-different-search'
+				)}
+				imgSrc={`${themeDisplay.getPathThemeImages()}/states/search_state.gif`}
+				small
+				title={Liferay.Language.get('no-results-found')}
+			/>
+		);
```

Comment:
Hi @BarbaraCabrera,
Do we really need to change the messages? Is it a change decided by a designer?